sycl/source/detail/kernel_impl.hpp

```diff
@@ -33,7 +33,7 @@ class kernel_impl {
 public:
   /// Constructs a SYCL kernel instance from a UrKernel
   ///
-  /// This constructor is used for plug-in interoperability. It always marks
+  /// This constructor is used for adapter interoperability. It always marks
```

Comment:
Does it make sense to say `UR adapter` instead of `adapter` for more clarity?

Reply:
agreed that does read a bit better